ansible/roles/batyam/meta/main.yml

```diff
@@ -0,0 +1,52 @@
+galaxy_info:
+  author: your name
+  description: your role description
+  company: your company (optional)
```

Comment:
company is Red Hat

Reply:
did some stuff here and decided stuff.. please confirm it's ok